internal/services/stats.go

```
@@ -195,6 +195,28 @@ func (s *Services) GetOverallStats(
 	}, nil
 }
 
+func (s *Services) GetStakerStats(
+	ctx context.Context, stakerPkHex string,
+) (*StakerStatsPublic, *types.Error) {
+	stats, err := s.DbClient.GetStakerStats(ctx, stakerPkHex)
+	if err != nil {
+		// Not found error is not an error, return nil
+		if db.IsNotFoundError(err) {
+			return nil, nil
+		}
```

Why not return an error? Seems a 4xx error to me